ui/components/app/confirm-page-container/enableEIP1559V2-notice/index.scss

```diff
@@ -0,0 +1,33 @@
+.enableEIP1559V2-notice {
+  &__message {
+    position: relative;
+
+    &__close-button {
+      background-color: transparent;
+      position: absolute;
+      right: 0;
+      top: 0;
+
+      &::after {
+        position: absolute;
+        content: '\00D7';
+        font-size: 29px;
+        font-weight: 200;
+        color: var(--black);
+        background-color: transparent;
+        top: 0;
+        right: 12px;
+        cursor: pointer;
+      }
+    }
+
+    a.enableEIP1559V2-notice__message__link {
+      @include H6;
+
+      padding: 0;
+      justify-content: flex-start;
+      font-weight: bold;
+      font-size: 0.75rem;
```

Comment:
font-size should be included in the H6 mixin and shouldn't need to be redeclared. Let me know if that isn't the case we'll want to mark that for further investigation.